klasses/views.py

```diff
@@ -0,0 +1,76 @@
+"""
+Views for Klasses app
+"""
+import logging
+
+from django.contrib.auth.models import User
+from rest_framework import (
+    authentication,
+    permissions,
+    status,
+)
+from rest_framework.generics import get_object_or_404
+from rest_framework.views import APIView
+from rest_framework.response import Response
+
+from backends.edxorg import EdxOrgOAuth2
+from klasses.api import get_user_klasses_info, get_user_klass_info
+from klasses.permissions import CanReadIfSelf
+
+log = logging.getLogger(__name__)
+
+
+class UserKlassList(APIView):
+    """
+    Class based view for user klasse list view.
```

Comment:
I see a lot of docstrings/comments that use this "klasse" spelling. Guessing that's a typo and it should be "klass"